src/commands/scorecard.js

```diff
@@ -33,15 +33,16 @@ export function getCurrentRepository() {
 export async function main(repo) {
   const result = typeof repo === "string" ? Ok(repo) : getCurrentRepository();
 
-  const repository = result.unwrap();
-
-  if (result.err) {
-    console.log(white().bold(result.val));
+  let repository;
```

Comment:
As `repository` could be potentially `undefined`, I wonder what happen for:
- this: https://github.com/NodeSecure/cli/pull/206/files#diff-5ef5720100afbb7a4704ea66e6e6cb42c713e78e2e6a0a858df3377c27f315f9R48
- and this: https://github.com/NodeSecure/cli/pull/206/files#diff-5ef5720100afbb7a4704ea66e6e6cb42c713e78e2e6a0a858df3377c27f315f9R54
- and this: https://github.com/NodeSecure/cli/pull/206/files#diff-5ef5720100afbb7a4704ea66e6e6cb42c713e78e2e6a0a858df3377c27f315f9R66

Reply:
For me it cannot be `undefined`
I mean if `unwrap()` throws then it close the process and we have this condition to check wether the repository exists, in `getCurrentRepository()`
```js
if (!rawPkg) {
    return Err("OSSF Scorecard supports projects hosted on Github only.");
}
```
`return Err()` will make `unwrap()` to throws.
Is it ok ?
I still can do `let repository = ""` so it will throws later (in case), but this case should not happen